lib/router/index.js

```diff
@@ -120,7 +121,7 @@ proto.param = function param(name, fn) {
 
   // ensure we end up with a
   // middleware function
-  if ('function' !== typeof fn) {
+  if (CONSTANTS.TYPES.FUNCTION !== typeof fn) {
```

Comment:
this is probably the only line that has switched placement of constant and typeof.
To maintain consistency, it should also be `if (typeof fn !== CONSTANTS.TYPES.FUNCTION) {`

Reply:
If others agree with you I will update it